product_docs/docs/migration_portal/3.5.2/01_mp_release_notes/index.mdx

```diff
@@ -0,0 +1,18 @@
+---
+title: "Release Notes"
+redirects:
+  - ../01_whats_new/
+---
+
+The Migration Portal documentation describes the latest version of Migration Portal.
+
+New features, enhancements, bug fixes, and other changes in Migration Portal 3.5.2 include the following:
+
+| Type | Description |
+| ---- |------------ |
+| Bug Fix | Fixed the `Too many open files` error that was causing frequent outages for the web-based version of Migration Portal. |
```

Comment:
Change the Description to:
"Fixed an issue that was causing frequent outages for the web-based version of Migration Portal."